taxonomies/food/ingredients.txt

```diff
@@ -54354,9 +54354,9 @@ hi: मुख्य जड़वाले सब्जी
 hr: korjenasto povrće s glavnim korijenom
 eurocode_2_group_2:en: 8.38
 
-# description:en:ROOT VEGETABLES are underground plant parts eaten by humans as food.
+# description:en:ROOT VEGETABLES are underground plant parts eaten by humans as food. This ingredient shouldn't be under vegetable since it can contain potato which is not a vegetable.
```

Comment:
Well potatoes are considered a vegetable in a lot of classification, like the Eurocode: https://web.archive.org/web/20170712020835/http://www.ianunwin.demon.co.uk/eurocode/docmn/ec99/ecmg08cl.htm#Listing

The Nutri-Score does not count them as fruits and vegetables, we use the Eurocodes for that.